spec/system/kudo_spec.rb

```diff
@@ -10,10 +10,12 @@
   let!(:employee) { create(:employee) }
 
   it 'crud kudo' do
-    user = FactoryBot.create(:employee)
-    login_as(employee)
-
     visit root_path
+    click_link 'Login page'
+    fill_in 'Email', with: employee.email
+    fill_in 'Password', with: employee.password
+    click_button 'Log in'
+    expect(page).to have_content 'Signed in successfully.'
```

Comment:
Consider using devise login helper here - you have already covered user logging with tests, so there is not necessary to check it here again.

https://webcache.googleusercontent.com/search?q=cache:prAqPB4X0REJ:https://sajadtorkamani.com/how-to-login-in-capybara-tests-when-using-devise/+&cd=3&hl=pl&ct=clnk&gl=pl

Reply:
I had a few rspec bugs in this round and applied some corrections, they are here for which I apologize.